Game crashes on pause

## Description
When in a game, pressing Esc causes the game to crash. 

```
Exception in thread "main" com.badlogic.gdx.utils.GdxRuntimeException: Couldn't load file: images/PauseMenu/PauseNew.jpg
        at com.badlogic.gdx.graphics.Pixmap.<init>(Pixmap.java:190)
        at com.badlogic.gdx.graphics.TextureData$Factory.loadFromFile(TextureData.java:101)
        at com.badlogic.gdx.graphics.Texture.<init>(Texture.java:122)
        at com.badlogic.gdx.graphics.Texture.<init>(Texture.java:114)
        at com.csse3200.game.components.gamearea.GameAreaDisplay.setPauseMenu(GameAreaDisplay.java:57)
        at com.csse3200.game.components.player.OpenPauseComponent.openPauseMenu(OpenPauseComponent.java:36)
        at com.csse3200.game.components.player.OpenPauseComponent.togglePauseMenu(OpenPauseComponent.java:30)
        at com.csse3200.game.events.EventHandler.lambda$trigger$0(EventHandler.java:85)
        at java.base/java.lang.Iterable.forEach(Iterable.java:75)
        at com.csse3200.game.events.EventHandler.forEachListener(EventHandler.java:152)
        at com.csse3200.game.events.EventHandler.trigger(EventHandler.java:85)
        at com.csse3200.game.components.player.KeyboardPlayerInputComponent.keyDown(KeyboardPlayerInputComponent.java:78)
        at com.csse3200.game.input.InputService.keyDown(InputService.java:93)
        at com.badlogic.gdx.InputEventQueue.drain(InputEventQueue.java:58)
        at com.badlogic.gdx.backends.lwjgl3.DefaultLwjgl3Input.update(DefaultLwjgl3Input.java:189)
        at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Window.update(Lwjgl3Window.java:378)
        at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.loop(Lwjgl3Application.java:192)
        at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:166)
        at com.csse3200.game.desktop.DesktopLauncher.main(DesktopLauncher.java:11)
Caused by: com.badlogic.gdx.utils.GdxRuntimeException: File not found: images/PauseMenu/PauseNew.jpg (Internal)
        at com.badlogic.gdx.files.FileHandle.read(FileHandle.java:142)
        at com.badlogic.gdx.files.FileHandle.readBytes(FileHandle.java:228)
        at com.badlogic.gdx.graphics.Pixmap.<init>(Pixmap.java:187)
        ... 18 more
```

## How To Reproduce
Steps to reproduce the behaviour:
1. Launch game
2. Start a new game
3. Press 'Esc'

## Expected Behaviour
4. Game displays pause menu

--- a/source/core/src/main/com/csse3200/game/components/gamearea/GameAreaDisplay.java
+++ b/source/core/src/main/com/csse3200/game/components/gamearea/GameAreaDisplay.java
@@ -54,7 +54,7 @@ public class GameAreaDisplay extends UIComponent {
     openPauseComponent = ServiceLocator.getGameArea().getPlayer().getComponent(OpenPauseComponent.class);
     Image pauseMenu;
         pauseMenu = new Image(new Texture(Gdx.files.internal
-            ("images/PauseMenu/PauseNew.jpg")));
+            ("images/PauseMenu/Pausenew.jpg")));
     pauseMenu.setSize(1300, 700);
     pauseMenu.setPosition((float) (Gdx.graphics.getWidth()/ (double) 2 - pauseMenu.getWidth()/2),
             (float) (Gdx.graphics.getHeight()/ (double) 2 - pauseMenu.getHeight()/2));

